src/constants.js

```diff
@@ -169,22 +178,32 @@ export const styles = {
   scrollableContentsTinted: {
     overflow: "hidden",
     borderRadius: 0,
-    backgroundColor: "rgb(206, 206, 206)",
+    backgroundColor: "#f2f2f2",
     padding: 10
   },
 
   scrollingContents: {
     overflow: "auto",
     height: "100%",
-    boxSizing: "border-box"
+    boxSizing: "border-box",
+    overflowWrap: "break-word"
   },
 
   contents: {
     borderRadius: 0,
-    backgroundColor: "rgb(206, 206, 206)",
+    backgroundColor: "#f2f2f2",
     padding: 15
   },
 
+  contentsCsvButton: {
+    borderRadius: 0,
+    backgroundColor: "#f2f2f2",
```

Comment:
This color is used a few time - worth extracting into a variable? I think we have a `colors` object already we could use, but I don't feel strongly about it.

Reply:
Done.